### ISSUE TITLE
popsplit problem

### ISSUE THREAD

**Issue by PsycoYT** [Megabyte918/MultiOgar-Edited#185]
The problem is that you should get too close to the virus

**Comment by PrinceDark9243** [comment]
yes your are right but i fixed it and made Good Popsplit for my sever MultiOgar 1.3.5

**Comment by NameYT** [comment]
@PrinceDark9243 Easy Popsplit? Long Distance like that give me download please :)

**Comment by Jhair2015** [comment]
@Megabyte918  To me it's Working the PopSplit, i record a video playing on my server with OGARio =)
